src/bitcoin-cli.cpp

```diff
@@ -293,6 +302,13 @@ int CommandLineRPC(int argc, char *argv[])
             argc--;
             argv++;
         }
+        std::string rpcPass;
+        if (GetBoolArg("-stdinrpcpass", false)) {
+            if(!std::getline(std::cin,rpcPass)) {
+                std::cerr<<"error: -stdinrpcpass specified but failed to read from standard input\n";
```

Comment:
Please use `throw std::runtime_error("-stdinrpcpass specified but failed to read from standard input")`, this will do automatically the right thing (like adding error: in front, adding a newline and setting the return code).